getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -40,7 +40,7 @@ Setting up the playable area
 
 We're going to create our main scene with a plain :ref:`Node <class_Node>` as its root. In the
 *Scene* dock, click the *Add Child Node* button represented by a "+" icon in the
-top-left and double-click on *Node*. Name the node ``Main``. Alternatively, to add
+top-left and double-click on *Node*. Name the node ``Main``. An alternate method to rename the node is to right-click on *Node* and choose :menuselection:`Rename` (or :kbd:`F2`). Alternatively, to add
```

Comment:
If this is a valid RST/Sphinx directive, we don't use it anywhere. Use asterisks for formatting instead.